dev-util/spirv-headers/spirv-headers-1.3.ebuild

```diff
@@ -0,0 +1,19 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit cmake-utils
+
+DESCRIPTION="Machine-readable files for the SPIR-V Registry"
+HOMEPAGE="https://www.khronos.org/registry/spir-v/"
+SNAPSHOT_COMMIT="02ffc719aa9f9c1dce5ce05743fb1afe6cbf17ea"
```

Comment:
Minor nit: I'd suggest using `EGIT_COMMIT` for consistency with what git-r3 uses.

Reply:
thanks. i dont have a strong opinion so i changed it